src/secp256k1/contrib/lax_der_parsing.c

```diff
@@ -5,7 +5,9 @@
  **********************************************************************/
 
 #include <string.h>
-#include <secp256k1.h>
+#ifdef HAVE_CONFIG_H
```

Comment:
I think you have to remove the `src/secp256k1` changes. It's a unchanged subtree (upstream is https://github.com/bitcoin-core/secp256k1).